com.unity.render-pipelines.universal/Runtime/ScriptableRendererFeature.cs

```diff
@@ -39,6 +39,14 @@ void OnValidate()
             Create();
         }
 
+        /// <summary>
+        /// Override this method and return true if the feature should use the Native RenderPass API
+        /// </summary>
+        public virtual bool SupportsNativeRenderPass()
```

Comment:
Can we make this internal until we actually enable the feature / discuss if this is final API?